### ISSUE TITLE
Enhancement: Notify User About Saved Image

### ISSUE THREAD

**Issue by abequette99** [info442-spr22/g-team#139]
Description: Add a message or pop up saying image was saved. When trying to save an image I had no idea if it was saved or not until I clicked on the gallery's page.

Explanation: It would be useful to have a small message saying go check out gallery to view saved image or a message saying saved because after I hit saved I was confused as to where it went or if it was saved.

**Comment by sharlenef** [comment]
Duplicate of #134 (#139, #155, #156, #171)
